platform/vs/sonic-version/sonic_version.yml.j2

```diff
@@ -0,0 +1 @@
+../../../files/build_templates/sonic_version.yml.j2
```

Comment:
Is there not a pre-existing env variable that defines the path ?

Reply:
This is a symbolic link or are you refering to https://github.com/Azure/sonic-buildimage/pull/6712/files/f11d60e07ba8cbee72a1aa9f83b4db70f1701b17#diff-b815aaa1a414e83000682ee2995e261dc33c72c4a2ff21771dc8ed1246416e53R485?
If yes, at build_debian.sh stage of build there is no BUILD_TEMPLATES environment variable that defines this path.